Builder/BuilderInterface.php

```diff
@@ -22,8 +22,11 @@
 interface BuilderInterface
 {
     /**
-     * @param AdminInterface            $admin
-     * @param FieldDescriptionInterface $fieldDescription
+     * Adds missing information to the given field description from the model manager metadata,
```

Comment:
@greg0ire  Yep I think you're right.
What trips me up is that `ModelManager::getNewFieldDescriptionInstance()` populates the FieldDescription with metadata, then `BuilderInterface::fixFieldDescription()` populates it again...